TUTORIAL.md

```diff
@@ -371,8 +371,7 @@ model:
       target_modules:
       - q_proj
       - k_proj
-    target_modules:
-    - 'Wqkv'
+      - 'Wqkv'
```

Comment:
oops, thanks for catching! However, `Wqkv` should be removed entirely as no model will have both `q_proj`/`k_proj` and `Wqkv` :)